public/main.js

```diff
@@ -0,0 +1,26 @@
+const randomNumber = Math.floor(Math.random() * (100) + 1)
+
+let counter = 0
+
+const guessNumber = (input) => {
+  let inputValue = input.value
+  console.log(randomNumber);
+  if (isNaN(inputValue) || inputValue < 1 || inputValue > 100) alert('Must be a number between 1 and 100')
+  if (inputValue == randomNumber) alert('You Win!')
```

Comment:
The `if` expressions below line 8 should be `else if` expressions because all of these cases are related to verifying whether or not `inputValue`is valid.